Code/GraphMol/MolAlign/AlignMolecules.cpp

```diff
@@ -30,7 +30,10 @@ double getAlignmentTransform(const ROMol &prbMol, const ROMol &refMol,
   if (atomMap == 0) {
     // we have to figure out the mapping between the two molecule
     MatchVectType match;
-    if (SubstructMatch(refMol, prbMol, match)) {
+    const bool recursionPossible = true;
+    const bool useChirality = false;
+    const bool useQueryQueryMatches = true;
```

Comment:
This piece isn't backwards compatible, but it's enough of an edge case that I think it's unlikely to be a problem.